### PR TITLE
Removed system probe creation and updated lib images

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: mkdocs/docs/experiments/categories/pods/pod-dns-error.md [hunk 1]
```diff
@@ -4,29 +4,26 @@
...
-    -  Ensure that the <code>pod-dns-error</code> experiment resource is available in the cluster by executing <code>kubectl get chaosexperiments</code> in the desired namespace. If not, install from <a href="https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/pod-dns-error/experiment.yaml">here</a>
+??? info "Verify the prerequisites" - Ensure that Kubernetes Version > 1.16 - Ensure that the Litmus Chaos Operator is running by executing <code>kubectl get pods</code> in operator namespace (typically, <code>litmus</code>).If not, install from <a href="https://v1-docs.litmuschaos.io/docs/getstarted/#install-litmus">here</a> - Ensure that the <code>pod-dns-error</code> experiment resource is available in the cluster by executing <code>kubectl get chaosexperiments</code> in the desired namespace. If not, install from <a href="https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/pod-dns-error/experiment.yaml">here</a>
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/pods/pod-dns-error.md:16]: why we removed newline here?

File: mkdocs/docs/experiments/categories/pods/pod-dns-error/target-hostnames.yaml [hunk 2]
```diff
@@ -12,13 +12,13 @@ spec:
...
+    - name: pod-dns-error
+      spec:
+        components:
+          env:
+            ## comma separated list of host names
+            ## if not provided, all hostnames/domains will be targeted
+            - name: TARGET_HOSTNAMES
+              value: '["litmuschaos"]'
+            - name: TOTAL_CHAOS_DURATION
+              value: "60"
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/pods/pod-dns-error/target-hostnames.yaml:24]: Can you check the below github action warnings?

File: mkdocs/docs/experiments/categories/pods/pod-dns-error.md (hunk 1 above)
Comment (Saranya-jena) [reply mkdocs/docs/experiments/categories/pods/pod-dns-error.md:16]: My vscode formatted it, reverting back the changes :)

File: mkdocs/docs/experiments/categories/pods/pod-dns-error/target-hostnames.yaml (hunk 2 above)
Comment (Saranya-jena) [reply mkdocs/docs/experiments/categories/pods/pod-dns-error/target-hostnames.yaml:24]: same :)

**Review by hrishavjha — APPROVED** [review]

**Review by SahilKr24 — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**